presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -131,6 +131,7 @@
     private final boolean reorderFilters;
     private final boolean enforceMemoryBudget;
 
+    private final List<Integer> outputColumns;
     private final OrcPredicate predicate;
```

Comment:
Is this predicate still used?

Reply:
Yes. It is still used to skip striped based on stats: https://github.com/prestodb/presto/blob/a5b230d56c6e010cfddc164ee850a0f36d7df13c/presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java#L244